### PR TITLE
Vmi/get access token prod 7756

### PR DESCRIPTION
**Pull request by jreynard-code**
Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [X] Testing
- [X] No error logs in web console (except iFrame errors)
- [X] Create or Update documentation (README file for users/integrators)
- [X] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature 
- [X] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

**Comment by jreynard-code** [comment]
@csm-thu @Nikeleos @COSMO-VMI  for this kind of dev, Environment variables should be initialised in WebApp definition (portal.azure).
If you want to test it locally, I can provide you the local.settings.json with functional parameters.

File: api/GetEmbedInfo/authentication.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const getAccessToken = async function () {
+  // Use ADAL.js for authentication
+  const adal = require('adal-node');
```
[the remark this answers is not in the document]
Comment (jreynard-code) [reply api/GetEmbedInfo/authentication.js]: done with luv'

File: api/GetEmbedInfo/authentication.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  const AuthenticationContext = adal.AuthenticationContext;
+
+  let authorityUrl = process.env.POWER_BI_AUTHORITY_URI;
+
+  authorityUrl = authorityUrl.replace('common', process.env.POWER_BI_TENANT_ID);
```
[the remark this answers is not in the document]
Comment (jreynard-code) [reply api/GetEmbedInfo/authentication.js]: done with luv'

**Review by csm-thu — APPROVED** [review]
Nice job ! :+1:

File: src/services/powerBI/embedConfigService.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+	const token = tokenResponse.accessToken;
+	return {
+		"Content-Type": "application/json",
+		Authorization: utils.getAuthHeader(token),
+	};
+}
+
+module.exports = {
+	getEmbedInfo: getEmbedInfo,
+};
```
Comment (Nikeleos) [inline src/services/powerBI/embedConfigService.js:196]: It seems to be indentation tabs errors.

**Pull request merged**